config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: %i[new create edit]
```

Comment:
Your routes should include the `update`. action. i.e.
`resources :users, only: [:new, :create, :edit, :update]`